pset-functions.js

```diff
@@ -208,6 +343,28 @@ console.log('Problem 5 - bonus:')
 
 // Add your code below this line
 
+function wordCount(phrase){
+  phrase = phrase.toLowerCase()
+  const words = phrase.split(" ")
+  let counter = {}
+
+  // remove lasta character of word if it punctuation
+  words.forEach((word) => {
+    if(word[word.length-1]=== '!' || word[word.length-1]=== '.' || word[word.length-1]=== ',' || word[word.length-1]=== '?'){
```

Comment:
Way to think ahead and keep edge cases in mind! Great use of forEach and if/else statements to feed the empty object with key/value pairs; very well done!

Overall, this is a very good job Katie! Continue to make some of the adjustments listed above and keep up the good work!